### DISCUSSION TITLE
Warning showing up in YAML section asking me to change properties

### DISCUSSION THREAD

**Discussion by CLaBruno** [SilentVoid13/Templater#1225] [Help] [answered]
I have a template setup with the following  in the YAML section:
<img width="497" alt="Screenshot 2023-10-13 004637" src="https://github.com/SilentVoid13/Templater/assets/8673832/2fdb5a71-30fe-490b-9152-21580e342cb2">Created:

But I am getting a warning saying that some of the data is incompatible with the property key. I tried changing the property type to date & time thinking that may be the issue but I am still getting the warning. Any ideas? Also, the date and time are not updating when edited on the last_modified property.

**Comment by Zachatoo** [comment, marked as answer]
> I am getting a warning saying that some of the data is incompatible with the property key.

It is incompatible, since it's not a date yet. I usually escape my frontmatter using `<% "---" %>` instead of `---` so I don't get that error in my template, but it still gets applied in the note I apply the template to.

```
<% "---" %>
Created: <% tp.file.creation_date() %>
Modified: <% tp.file.modified_date() %>
<% "---" %>
```

> Also, the date and time are not updating when edited on the last_modified property.

That is expected. They don't update, they are executed once when the template is executed, and then it's static plain text.

If you want created/modified dates in frontmatter to auto update, I highly recommend the Linter plugin for that use case.
